doc/source/user_guide/boolean.rst

```diff
@@ -37,6 +37,18 @@ If you would prefer to keep the ``NA`` values you can manually fill them with ``
 
    s[mask.fillna(True)]
 
+If you create a column of ``NA`` values (for example to fill them later)
+with ``df['new_col'] = pd.NA``, the ``dtype`` would be set to ``object`` in the
+new column. The performance on this column will be worse than with
+the appropriate type. It's better to use
+``df['new_col'] = pd.Series(pd.NA, dtype='Int64')``
```

Comment:
```suggestion
``df["new_col"] = pd.Series(pd.NA, dtype="boolean")``
```
since this note is for Boolean dtype.

Also, I think double-quoted string is preferred, for code consistency.

Reply:
Changed, thanks!